MovieCatalogCrawler/src/main/java/moviecrawler/dtos/MovieDTO.java

```diff
@@ -6,16 +6,12 @@
 public class MovieDTO {
```

Comment:
Implement equals and hashcode methods since we are using the equals method to filter movies that already exists at the moment.

Reply:
The filtring is made by Movie Entity, and so the MovieDTO has no need for equals and hashcode.